packages/@aws-cdk/aws-appmesh/test/test.route.ts

```diff
@@ -188,6 +239,10 @@ export = {
           match: {
             prefixPath: '/node',
           },
+          timeout: {
+            idle: cdk.Duration.seconds(10),
+            perRequest: cdk.Duration.seconds(10),
+          },
```

Comment:
Same note in this file - only add the `timeout` property if you're actually asserting it in the test. If you're not, leave the test unchanged.

Reply:
yes missed this one, adding the assertion.